content/en/docs/contribute/localization.md

```diff
@@ -97,9 +118,9 @@ Provide guidance to localization contributors in the localized `README-**.md` fi
 - A point of contact for the localization project
 - Any information specific to the localization
 
-After you create the localized README, add a link to the file from the main English file, [`README.md`'s Localizing Kubernetes Documentation] and include contact information in English. You can provide a GitHub ID, email address, [Slack channel](https://slack.com/), or other method of contact.
+After you create the localized README, add a link to the file from the main English file, [`README.md`'s Localizing Kubernetes Documentation] and include contact information in English. You can provide a GitHub ID, email address, [Slack channel](https://slack.com/), or other method of contact. You must also provide a link to your localized Community Code of Conduct.
```

Comment:
Just noticed this, should the `README.md's Localizing Kubernetes Documentation` link to something, e.g. maybe it is good to link to the English README. Otherwise we can safely remove the `[` & `]` characters ;)

Reply:
@rlenferink Thanks--I tweaked it.